src/renderer/components/MainSideBar/Hide.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import React from "react";
 import { Transition } from "react-transition-group";
+import Box from "~/renderer/components/Box";
```

Comment:
I don't really mind the order of the imports, but could we at least group them by 
- libs first
- our own code after

Reply:
PROTIP https://github.com/amatiasq/vsc-sort-imports with:

```
"sort-imports.on-save": false,
"sort-imports.default-sort-style": "module-scoped",
```